### PR TITLE
UIQM-415: Create original bib record in quickMARC UI

### PR DESCRIPTION
**Pull request by daniil-patckin**
## Issue
[UIQM-415](https://issues.folio.org/browse/UIQM-415)

## Screencast

https://user-images.githubusercontent.com/124583895/230956317-d9228176-74cf-4e70-bd6d-bb2be971e09f.mov

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/QuickMarcEditor/utils.js [hunk 1]
```diff
@@ -167,6 +168,49 @@ const getCreateMarcRecordDefaultFields = (instanceRecord) => {
...
+    '008': HOLDINGS_FIXED_FIELD_DEFAULT_VALUES,
+    '009': '$a ',
+    '010': '2',
+    '011': '2',
+    '017': 'u',
+    '018': 'u',
+    '019': '\\',
+    '245': '$a ',
+    '999': '$s ',
+  };
```
Comment (Dmytro-Melnyshyn) [inline src/QuickMarcEditor/utils.js:186]: Are these initial values agreed with Christine?

**Comment by Dmytro-Melnyshyn** [comment]
1. After clicking the close `X` icon, the url will change from `/inventory/quick-marc/create-bib?sort=title` to `/inventory/view/?sort=title`. `/view/` shouldn't be there.
2. Please check the LDR set, it is not the same as described in the story.
3. Can't create a bib record with the same steps as in the screencast above.

https://user-images.githubusercontent.com/77053927/231153843-692cb33f-cab7-482e-b1c6-6f90679785ad.mp4

File: src/QuickMarcEditor/QuickMarcCreateBibWrapper.test.js [hunk 2]
```diff
@@ -23,47 +23,115 @@ jest.mock('react-final-form', () => ({
...
 const mockFormValues = jest.fn(() => ({
-  fields: undefined,
-  externalHrid: 'in00000000022',
-  externalId: '17064f9d-0362-468d-8317-5984b7efd1b5',
-  leader: '00000nu\\\\\\2200000un\\4500',
-  marcFormat: 'HOLDINGS',
-  parsedRecordDtoId: '1bf159d9-4da8-4c3f-9aac-c83e68356bbf',
-  parsedRecordId: '1bf159d9-4da8-4c3f-9aac-c83e68356bbf',
-  records: [
+  'externalId': '00000000-0000-0000-0000-000000000000',
```
Comment (BogdanDenis) [inline src/QuickMarcEditor/QuickMarcCreateBibWrapper.test.js:26]: Usually we don't use quotes for object keys. Please update it to follow current pattern

File: src/QuickMarcEditor/QuickMarcEditorContainer.js [hunk 3]
```diff
@@ -98,8 +98,8 @@ const QuickMarcEditorContainer = ({
       ? EXTERNAL_INSTANCE_APIS[MARC_TYPES.BIB]
       : EXTERNAL_INSTANCE_APIS[marcType];
 
-    const instancePromise = mutator.quickMarcEditInstance.GET({ path: `${path}/${externalId}` });
-    const marcRecordPromise = action === QUICK_MARC_ACTIONS.CREATE
+    const instancePromise = action === QUICK_MARC_ACTIONS.CREATE_BIB ? Promise.resolve({}) : mutator.quickMarcEditInstance.GET({ path: `${path}/${externalId}` });
```
Comment (BogdanDenis) [inline src/QuickMarcEditor/QuickMarcEditorContainer.js:101]: Please format it like so:
```
const instancePromise = action === QUICK_MARC_ACTIONS.CREATE_BIB
  ? Promise.resolve({})
  : mutator.quickMarcEditInstance.GET({ path: `${path}/${externalId}` });
```

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

**Pull request closed without merging**